### DISCUSSION TITLE
null items_list in received_documents

### DISCUSSION THREAD

**Discussion by marco-bozzo** [fattureincloud/api#36] [Q&A] [answered]
Ciao, richiamando l'url "https://api-v2.fattureincloud.it/c/xxxxxx/received_documents/$id" con il giusto companyId al posto delle xx e l'id del documento contenuto nella variabile $id, e con 'fieldset' => 'detailed', ottengo tutte le informazioni della fattura ricevuta ma con la lista degli articoli "items_list" nulla, se però guardo lo stesso documento da FattureInCloud vedo correttamente 1 articolo.

**Comment by valmoz** [comment]
Ciao @marco-bozzo,
è difficile darti supporto senza conoscere i dettagli del tuo account.
ti consiglio di provare a contattare il supporto clienti, in modo da poter passare in una conversazione privata dettagli come il company id e l'id del documento.

Grazie
Mauro

**Comment by marco-bozzo** [comment, marked as answer]
Ho sentito il supporto clienti ed è in corso la realizzazione della nuova registrazione acquisti che garantirà la lettura automatica degli articoli e la relativa registrazione; in questo modo si dovrebbero trovare gli articoli nella items_list.
